src/modules/NtpClient.rb

```diff
@@ -867,6 +861,20 @@ def pool_servers_for(known_countries)
     def cache_server(server)
       @ntp_servers[server["address"].to_s] = server
     end
+
+    def unsupported_error(unsupported)
+      msg = format(
+        # TRANSLATORS: error report. %s stands unsupported keys.
+        _("Ignoring the NTP configuration. The profile format has changed in an " \
+          "incompatible way. These keys are no longer supported: '%s'."),
+        unsupported.join("', '")
+      )
+
+      displayinfo = Yast::UI.GetDisplayInfo
+      width = displayinfo["TextMode"] ? displayinfo.fetch("Width", 80) : 80
```

Comment:
what about reusing https://github.com/yast/yast-yast2/blob/master/library/general/src/lib/ui/text_helpers.rb#L33 ?

Reply:
ah, now I see it. So makes sense. Maybe whole this display info changes makes sense to have in text helpers.